test/torchaudio_unittest/rnnt/rnnt_loss_cuda_test.py

```diff
@@ -0,0 +1,10 @@
+import torch
+from .rnnt_loss_impl import RNNTLossTest
+from torchaudio_unittest import common_utils
+from .utils import skipIfNoTransducer
+
+
+@skipIfNoTransducer
```

Comment:
nit: just realized the name has not been standardized from Transducer to RNNT, but this can be changed after this PR